torch/lib/TensorLib/function_wrapper.py

```diff
@@ -0,0 +1,363 @@
+import re
```

Comment:
Implements multiple-passes to take yaml and generate declarations and definitions. The two main functions are:

- create_generic: create the declaration, syntax sugar
- create_derived: create the definition for all types and processors